extensions/2.0/Vendor/EXT_instance_features/schema/featureId.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "featureId.schema.json",
+    "title": "Feature ID in EXT_instance_features",
+    "type": "object",
+    "description": "Feature IDs stored in a GPU mesh instanting attribute",
+    "allOf": [
+        {
+            "$ref": "glTFProperty.schema.json"
+        }
+    ],
+    "properties": {
+        "featureCount": {
+            "type": "integer",
+            "minimum": 1,
+            "description": "The number of unique features in the attribute."
+        },
+        "nullFeatureId": {
+            "type": "integer",
+            "minimum": 0,
+            "description": "A value that indicates that no feature is associated with this instance."
+        },
+        "attribute": {
+            "description": "An attribute containing feature IDs. When this is omitted, then the feature IDs are assigned to the GPU instances by their index.",
```

Comment:
@lilleyse is this intentional that GPU instaces can have this implicit range (essentially `constant: 0, repeat: 1`) but primitives _require_ an attribute? (even though the vertex ID could be used)

Reply:
Short term though, I'm good with the vertex ID approach. It's not as expressive as implicit feature ids were before it's easy to understand. Having some implicit feature ID capability is nice, especially once we start storing vector data in glTF. It's much more likely that GIS point features will have string and boolean properties, something that `propertyAttributes` doesn't support.

Reply:
I updated the schema to allow for per-vertex implicit feature IDs. https://github.com/CesiumGS/glTF/pull/56/commits/12f2d6396a346f8fbdb51fb388c81df0cf03b6d6

@javagl could you update the `EXT_mesh_features` text to account for this change?

Reply:
The PR is at https://github.com/CesiumGS/glTF/pull/57

Reply:
Thanks, merged!